reducers/todoApp.js

```diff
@@ -1,9 +1,68 @@
-const initialState = {
-  data: []
+import { ActionsTypes } from "../actions/actionTypes";
+import { combineReducers } from "redux";
+
+const initialStateHome = {
+  todo: [
+    {
+      id: 1,
+      title: "Description",
+      description: "Second Description",
+      completed: false
+    },
+    {
+      id: 2,
+      title: "Description",
+      description: "Second Description",
+      completed: false
+    },
+    {
+      id: 3,
+      title: "Description",
+      description: "Second Description",
+      completed: false
+    },
+    {
+      id: 4,
+      title: "Description",
+      description: "Second Description",
+      completed: true
+    },
+    {
+      id: 5,
+      title: "Description",
+      description: "Second Description",
+      completed: false
+    }
+  ]
 };
 
-function todoApp(state = initialState, action) {
-  return state;
+function homeReducer(state = initialStateHome, action) {
+  switch (action.type) {
+    case ActionsTypes.CLEAR_ALL_DONE:
+      return Object.assign({}, state, {
+        todo: state.todo.map((item, index) => {
+          return Object.assign({}, item, {
```

Comment:
I think we are using too much `Object.assign` where it's not that necessary.
For this case we could just use `map` once:
```js
return {
  todo: state.todo.map((t) => {
    t.completed = false;
    return t
  })
}
```